android/src/main/java/com/reactnativecommunity/slider/ReactSlider.java

```diff
@@ -8,7 +8,7 @@
 
 import android.content.Context;
 import android.os.Build;
-import android.support.v7.widget.AppCompatSeekBar;
+import androidx.appcompat.app.AppCompatSeekBar;
```

Comment:
Shouldn't it be `androidx.appcompat.widget.AppCompatSeekBar`?

Reply:
Yes! Good catch!

Reply:
Is this live yet? Even though it's been merged I still had `import androidx.appcompat.app.AppCompatSeekBar;` in my project